site/assignments/tsp.rst

```diff
@@ -0,0 +1,69 @@
+******************************
+Travelling Salesperson Problem
+******************************
+
+* **Maximum Points**: 30
+* **DUE**: Monday October 9, 2023 at 11:55pm; submitted on MOODLE.
+* **Files**: `Files available on the GitHub repository <https://github.com/jameshughes89/cs4XX-EvolutionaryComputation/tree/main/resources/tsp>`_
```

Comment:
could also tar/zip these up as part of the build

Reply:
I thought about it, but figured it was just simpler this way. Maybe I'll go back and add this once a version of the assignment is up and runnig.